pkg/nozzle/okta/okta.go

```diff
@@ -62,6 +63,16 @@ func (Driver) New(opts map[string]string) (nozzle.Nozzle, error) {
 		return nil, fmt.Errorf("okta nozzle requires 'subdomain' config parameter")
 	}
 
+	// check if operator entered full domain instead of subdomain
+	if strings.Contains(subdomain, ".okta.com") {
```

Comment:
This check may not work if the domain has .okta.com in the middle of it. We have a function in the Util package `util.ValidateURLSuffix` designed to check for this at the end of urls. can you swap to that and give it a test?